notify_python35.cpp

```diff
@@ -220,20 +213,20 @@ bool NotifyPython35::reconfigure(const std::string& newConfig)
 	// Configuration change is protected by a lock
 	lock_guard<mutex> guard(m_configMutex);
 
-	if (Py_IsInitialized())
-	{
-		// Cleanup Loaded module first
-		Py_CLEAR(m_pModule);
-		m_pModule = NULL;
-		Py_CLEAR(m_pFunc);
-		m_pFunc = NULL;
-		m_pythonScript.clear();
+	PyGILState_STATE state = PyGILState_Ensure(); // acquire GIL
 
-		// Remove Interpreter
-		Py_Finalize();
-		// Start interpreter, setting paths etc
-		interpreterStart(this);
-	}
+	// Reimport module
+	PyObject* newModule = PyImport_ReloadModule(m_pModule);
```

Comment:
Check return value of PyImport_ReloadModule. And if this call fails, should we clear m_pModule & m_pFunc?